src/main/scala/Player.scala

```diff
@@ -0,0 +1,18 @@
+import scala.util.Random
+
+object Player {
+  val random: Random.type = scala.util.Random
+  var health: Int =  random.nextInt(20)
+  var attacks: Int =  random.nextInt(20)
+  var hit: Int = random.nextInt(10)
+  var initiative: Int =  random.nextInt(20)
+  var armour: Int = random.nextInt(5)
+  var bonusArmour = 0
+  def Reroll(){
```

Comment:
method names start with a lowercase word, only words after should have capitalised first letter